botx/client/stickers_api/delete_sticker.py

```diff
@@ -0,0 +1,47 @@
+from uuid import UUID
+
+from botx.client.authorized_botx_method import AuthorizedBotXMethod
+from botx.models.api_base import UnverifiedPayloadBaseModel, VerifiedPayloadBaseModel
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+class BotXAPIDeleteStickerRequestPayload(UnverifiedPayloadBaseModel):
+    sticker_pack_id: UUID
+    sticker_id: UUID
+
+    @classmethod
+    async def from_domain(
+        cls,
+        sticker_pack_id: UUID,
+        sticker_id: UUID,
+    ) -> "BotXAPIDeleteStickerRequestPayload":
+        return cls(sticker_pack_id=sticker_pack_id, sticker_id=sticker_id)
+
+
+class BotXAPIDeleteStickerResponsePayload(VerifiedPayloadBaseModel):
+    status: Literal["ok"]
+
+
+class DeleteStickerMethod(AuthorizedBotXMethod):
+    async def execute(
+        self,
+        payload: BotXAPIDeleteStickerRequestPayload,
+    ) -> None:
+        sticker_pack_id = payload.sticker_pack_id
+        sticker_id = payload.sticker_id
+        path = f"/api/v3/botx/stickers/packs/{sticker_pack_id}/stickers/{sticker_id}"
+
+        response = await self._botx_method_call(
+            "POST",
+            self._build_url(path),
+            json=payload.jsonable_dict(),
```

Comment:
Wait. There are no json payload for this method. All goes in query params

Reply:
Not in query params, but just in url. Anyway it should be fixed, I agree with u.